javascript/src/seedsource/containers/Map.jsx

```diff
@@ -52,12 +52,13 @@ class Map extends React.Component {
         this.mapIsMoving = false
         this.shapefile = null
         this.geojson = null
+        this.simple = props.simple || false
     }
 
     // Initial map setup
     componentDidMount() {
         let lethargy = new Lethargy(7, 10, 0.05)  // Help minimize jumpy zoom with Apple mice and trackpads
-        L.Map.ScrollWheelZoom.prototype._onWheelScroll = function(e) {
+        L.Map.ScrollWheelZoom.prototype._onWheelScroll = function (e) {
```

Comment:
Please remove the added space between `function` and `()` here and on line 87. Or better yet, convert these to arrow functions.